src/tinypeel/tinypeel.py

```diff
@@ -573,6 +594,8 @@ def main():
     args.pedigree = args.ped_file
     args.startsnp = args.start_snp
     args.stopsnp = args.stop_snp
+    args.altalleleprobfile = args.alt_allele_prob_file
+    args.mainmetafounder = args.main_metafounder
```

Comment:
These two can be removed. I know these lines of code are a bit confusing, but they are just a temporary solution to the mismatch of the argument names in AlphaPeel and tinyhouse. As long as all of our operations are all in AlphaPeel, we don't need this.